k8s/aat/money-claims/legal-frontend/legal-frontend.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: cmc-legal-frontend
+  namespace: money-claims
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:latest-*
+spec:
+  releaseName: cmc-legal-frontend
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: cmc-legal-frontend
+    version: 0.0.1
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "4d7922fc-7d37-4278-b542-6ac8fe0408e2"
+      image: hmcts.azurecr.io/hmcts/cmc-legal-frontend:latest
+      ingressHost: cmc-legal-frontend.service.core-compute-aat.internal
+      keyVaults:
+        cmc:
+          resourceGroup: cmc
+          secrets:
+          - AppInsightsInstrumentationKey
+          - legal-oauth-client-secret
+          - cmc-s2s-secret
+          - legal-draft-store-primary
+          - legal-draft-store-secondary
+          - legal-cookie-encryption-key
+      environment:
+        IDAM_URL: https://preprod-idamapi.reform.hmcts.net:3511
+        IDAM_AUTHENTICATION_WEB_URL: https://idam.preprod.ccidam.reform.hmcts.net
+        PAY_URL: http://payment-api.service.core-compute-aat.internal
```

Comment:
change to internal service urls example : http://payment-api-java.fees-pay

Reply:
oh yeah!!!! sorry bad copy n paste